### PR TITLE
Use a method to generate the secondary target string

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by mpirri — APPROVED** [review]
👍

File: fastlane_core/lib/fastlane_core/update_checker/update_checker.rb [hunk 1]
```diff
@@ -186,14 +198,7 @@ def self.p_hash(args, gem_name)
...
       end
-
-      if app_identifier
-        return Digest::SHA256.hexdigest("p#{app_identifier}fastlan3_SAlt") # hashed + salted the bundle identifier
-      end
-
-      return nil
-    rescue
-      return nil
+      app_identifier
```
Comment (taquitos) [inline fastlane_core/lib/fastlane_core/update_checker/update_checker.rb]: I would add an explicit return here, just to make it clear you intend the output to be used

File: fastlane_core/lib/fastlane_core/update_checker/update_checker.rb [hunk 2]
```diff
@@ -342,13 +351,17 @@ def self.event_for_install_method(tool, ci, install_method, timestamp_seconds)
...
-          detail: ci
+          detail: secondary_target_string(ci)
         },
         millis_since_epoch: timestamp_seconds * 1000,
         version: 1
       }
     end
 
+    def self.secondary_target_string(string)
+      return string
```
Comment (taquitos) [inline fastlane_core/lib/fastlane_core/update_checker/update_checker.rb]: I take it this will be overridden?

Comment (ohayon) [inline fastlane_core/lib/fastlane_core/update_checker/update_checker.rb]: Yeasir!

File: fastlane_core/lib/fastlane_core/update_checker/update_checker.rb (hunk 1 above)
Comment (ohayon) [inline fastlane_core/lib/fastlane_core/update_checker/update_checker.rb]: ah, good point!

**Review by taquitos — APPROVED** [review]
🎈 🐐

**Pull request merged**